gitops-application.go

```diff
@@ -109,26 +127,28 @@ func createGitOpsApplicationPayload(requestBody map[string]interface{}) GitOpsAp
 	return newApplication
 }
 
-func createGitOpsApplicationPUTPayload(requestBody map[string]interface{}) Application {
-	Application := Application{
-		Metadata: Metadata{
-			Labels: Labels{
-				Envref:     valueToString(GetNestedValue(requestBody, "gitops", "application", "metadata", "labels", "harness.io/envRef").(string)),
-				Serviceref: valueToString(GetNestedValue(requestBody, "gitops", "application", "metadata", "labels", "harness.io/serviceRef").(string)),
-			},
-			Namespace: valueToString(GetNestedValue(requestBody, "gitops", "application", "metadata", "namespace").(string)),
-		},
-		Spec: Spec{
-			Source: Source{
-				RepoURL:        valueToString(GetNestedValue(requestBody, "gitops", "application", "spec", "source", "repoURL").(string)),
-				Path:           valueToString(GetNestedValue(requestBody, "gitops", "application", "spec", "source", "path").(string)),
-				TargetRevision: valueToString(GetNestedValue(requestBody, "gitops", "application", "spec", "source", "targetRevision").(string)),
+func createGitOpsApplicationPUTPayload(requestBody map[string]interface{}) GitOpsApplication {
+	putApp := GitOpsApplication{
```

Comment:
nit: We can change the variable name to improve readability. Ex: `GitOpsApplicationData` / `GitopsAppDetails` (these are just examples, feel free to rename whatever is readable acc to you)

Reply:
+1